Tweaks to event handler documentation (#496)
cylc/cylc-doc: src/user-guide/writing-workflows/runtime.rst

   succeeded                                  job succeeded
   submission timeout                         job timed out in the ``submitted`` state
   execution timeout                          job timed out in the ``running`` state
   warning                                    scheduler received a message of severity WARNING from job
   critical                                   scheduler received a message of severity CRITICAL from job

>    custom                                     scheduler received a message of severity CUSTOM from job |br| (note: Literally of severity ``CUSTOM``, not any user defined severity level.)

Reviewer: ```suggestion
   custom                                     scheduler received a message of severity CUSTOM from job |br| (note: literally, the word ``CUSTOM``)
```
